src/hooks/useInputScroll.ts

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+
+const useInputScroll = (ref: React.RefObject<HTMLInputElement>) => {
+  useEffect(() => {
+    window.addEventListener('touchmove', handleScroll);
+    return () => {
+      window.removeEventListener('touchmove', handleScroll);
+    };
+  }, []);
+
+  const handleScroll = (e: { target: any }) => {
+    if (
+      document.activeElement == ref.current ||
+      ref.current?.contains(e.target)
+    ) {
+      (document.activeElement as HTMLElement).blur();
+    }
+  };
+};
+
+export default useInputScroll;
```

Comment:
`e`를 파라미터로 넘길 때 destructuring 할 필요는 없을 것 같아요! 

```suggestion
import React, { useEffect } from 'react';

const useInputScroll = (ref: React.RefObject<HTMLInputElement>) => {
  useEffect(() => {
    window.addEventListener('touchmove', handleScroll);
    return () => {
      window.removeEventListener('touchmove', handleScroll);
    };
  }, []);

  const handleScroll = (e: TouchEvent) => {
    if (
      document.activeElement === ref.current ||
      ref.current?.contains(e.target as Node)
    ) {
      (document.activeElement as HTMLElement).blur();
    }
  };
};

export default useInputScroll;

```